### PR TITLE
Set a smart default value for property-bound colors.

### PR DESCRIPTION
**Pull request by simeoncran**
Previously we always set property-bound colors to red to make them visible. With this change we are a little smarter about it: we show the color the designer used, and if that color was animated we find a high-alpha color amongst the keyframes the designer used.

### CODE REVIEW

File: source/LottieToWinComp/LottieToWinCompTranslator.cs [hunk 1]
```diff
@@ -2805,6 +2805,18 @@ static string FindFirstBindingNameForProperty(string bindingString, string prope
                 inheritedOpacity.ComposedWith(context.TrimAnimatable(shapeFill.Opacity)),
                 bindingSpec: shapeFill.Name);
 
+        // Returns a single color that can be used to represent the given animatable color.
+        // This is used as the default color for property bindings. If the animatable color is
+        // not animated then we return its value. If it's animated we return the value of the
+        // keyframe with the highest alpha, so that it likely to be visible.
```
Comment (derobinsMSFT) [inline source/LottieToWinComp/LottieToWinCompTranslator.cs]: >it [](start = 52, length = 2)

"it's [more]"

File: source/LottieToWinComp/LottieToWinCompTranslator.cs [hunk 2]
```diff
@@ -2805,6 +2805,18 @@ static string FindFirstBindingNameForProperty(string bindingString, string prope
...
+        // The actual color we return here isn't all that important as it is expected to be set
```
Comment (derobinsMSFT) [inline source/LottieToWinComp/LottieToWinCompTranslator.cs]: Nit: add a comma here.

Comment (simeoncran) [reply source/LottieToWinComp/LottieToWinCompTranslator.cs]: I can't tell where on this line you are referring to.

---
In reply to: [373595653](https://github.com/windows-toolkit/Lottie-Windows/pull/233#discussion_r373595653) [](ancestors = 373595653)

File: source/LottieToWinComp/LottieToWinCompTranslator.cs [hunk 3]
```diff
@@ -2805,6 +2805,18 @@ static string FindFirstBindingNameForProperty(string bindingString, string prope
...
+        // to some other value at runtime via property binding, but it is handy to have a visible
+        // color when testing, and even better if the color looks like what the designer saw.
+        static Color DefaultValueOf(Animatable<Color> animatableColor)
+            => animatableColor.IsAnimated
+                ? animatableColor.KeyFrames.ToArray().OrderByDescending(kf => kf.Value.A).Select(kf => kf.Value).First()
```
Comment (derobinsMSFT) [inline source/LottieToWinComp/LottieToWinCompTranslator.cs]: >OrderByDescending(kf => kf.Value.A).Select(kf => kf.Value).First() [](start = 54, length = 66)

Cleaner: `OrderByDescending(kf => kf.Value.A).First().Value`

**Pull request merged**